components/admin/questDetails.tsx

```diff
@@ -238,12 +239,16 @@ const AdminQuestDetails: FunctionComponent<QuestDetailsProps> = ({
           </Button>
         </div>
         {isEdit && (
```

Comment:
Line 230, could you edit ` <div className="w-full flex justify-center gap-8">` by adding `flex-wrap` to the className please ? Because there is not enough space for the buttons